### DISCUSSION TITLE
Try to deploy g4f via docker provider. But nothing happened.

### DISCUSSION THREAD

**Discussion by boh5** [Dokploy/dokploy#729] [General]
Provider setting:
![image](https://github.com/user-attachments/assets/14235a4d-afc1-493d-ad7a-cba2dbb842e9)

The deployment shows that it just pulls the image, nothing else happened:
![image](https://github.com/user-attachments/assets/6fcb2eb5-6113-4fd4-801a-45e514742b00)

No container is running, even no container is built:
![image](https://github.com/user-attachments/assets/5c39273f-6a01-4859-88d1-7219b41156b0)

And I have tried to deploy other services via docker provider, but nothing happened either. I have searched the docs, but found nothing helpful.
So the question is "how to deploy a service via docker provider?"

**Comment by boh5** [comment]
I found the problem that my volume configuration is incorrect. However, I cannot see any error information on the service page or deployments page. I found the problem in the `Home -> Settings -> Server -> Web Server -> Server -> Watch logs` popup. If the error information were displayed on the service page, the user experience might be better.

**Comment by Siumauricio** [comment]
Read here: https://docs.dokploy.com/docs/core/troubleshooting#mounts-are-causing-my-application-not-to-run

If you add a invalid mount even if your build of your application were successful it will not display the error

**Reply by boh5** [reply]
Yes, it's my fault. But if this error message could be displayed in the deployment, that would be great.
THX
